layouts/try/baseof.html

```diff
@@ -49,8 +49,28 @@ <h2>Interactive Tutorials</h1>
                 <div class="row">
                     <div class="col-md-4 pl-1 pr-1 pb-3">
                         <article class="card card-custom-style interactive-tutorial-trigger"
-                            style="background-color: hsl(171, 100%, 41%)" data-katacoda-src="ncskier/tekton-dashboard"
-                            data-github-lnk="ncskier/katacoda/tree/master/tekton-dashboard" data-qwiklabs-lnk="">
+                            style="background-color: hsl(171, 100%, 41%)" data-katacoda-src="ratrosyu/playground"
+                            data-github-lnk="tektoncd/website/tutorials/katacoda/tree/master/" data-qwiklabs-lnk="">
+                            <div class="card-img-top">
+                                <img src="/try/icons/orgainize.png" class="img-fluid">
+                            </div>
+                            <div class="card-body card-body-custom-style">
+                                <p class="h4 text-white">Play with Tekton in the playground</p>
+                            </div>
+                        </article>
+                        <article class="card card-custom-style interactive-tutorial-trigger"
+                            style="background-color: hsl(171, 100%, 41%)" data-katacoda-src="tektoncd/basic-pipeline"
+                            data-github-lnk="tektoncd/website/tutorials/katacoda/tree/master/basic-pipeline" data-qwiklabs-lnk="">
```

Comment:
should this be `tektoncd/website/tree/master/tutorials/katacoda/basic-pipeline`?